src/Umbraco.Abstractions/Models/ImageFileModel.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Umbraco.Core.Models
+{
+    public class ImageFileModel
+    {
+        public byte[] Data { get; }
```

Comment:
This is an incredibly allocation heavy pattern.

Reply:
Thank you @JimBobSquarePants! The implementation will be refactored